views/js/runner/plugins/tools/magnifier/magnifierPanel.js

```diff
@@ -112,11 +124,14 @@ define([
         var observer = null;
         var targetWidth, targetHeight, dx, dy;
         var scrolling = [];
+        var dynamicComponentInstance;
+
+        var dynamicComponentConfig = _.defaults(dynamicComponentConfig || {}, dynamicComponentDefaultConfig);
 
         /**
          * @typedef {Object} magnifierPanel
          */
-        var magnifierPanel = movableComponent({
+        var magnifierPanel = component({
```

Comment:
Was it not possible to use only one dynamic component instead?

Reply:
Did an investigation and found out that it is worth to move this requirement to separate task because of complexity and amount of code .

Accordingly your approve, I will create a task shortly